spec/helpers/case_workers/claims_helper_spec.rb

```diff
@@ -112,4 +112,68 @@
       expect(helper.claim_count).to eq 3
     end
   end
+
+  describe '#format_miscellaneous_fee_names' do
+    subject { format_miscellaneous_fee_names(claim) }
+
+    let(:claim) { create(:claim) }
+
+    context 'When there are no eligible fee types' do
+      before do
+        allow(claim).to receive(:eligible_misc_fee_types).and_return([])
+      end
+
+      it { is_expected.to be_empty }
+    end
+
+    context 'When there are brackets in the fee type description' do
+      let(:fee_type) do
+        [instance_double(Fee::MiscFeeType, description: 'Abuse of process hearings (half day)')]
+      end
+
+      before do
+        allow(claim).to receive(:eligible_misc_fee_types).and_return(fee_type)
+      end
```

Comment:
For a small cleanup, this can go outside of the contexts as they are identical in most of the contexts. If you decide to do this then for the 'no eligible fee types' context you will need to add `let(:fee_type) { [] }`.